src/main/resources/application-example.yml

```
@@ -1 +1,24 @@
+jwt:
+  secret-key: EXAMPLE7A3E4F37B3DAD9CD8KEY6AA4B1AF7123!@#
+  access-token:
+    expiration-time: 600000
```

```suggestion
    expiration-time: 600000 #(ms) = 10 minutes
```
처럼 바로 몇 초인지 알 수 있도록 하는 건 어떨까요?

---

좋습니다 👍